CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Flutter Plugin Changelog
 
+## Version 9.1.1 - December 9, 2024
+
+Patch release that updates the Android SDK to [18.5.1](https://github.com/urbanairship/android-library/releases/tag/18.5.1) and the iOS SDK to [18.13.1](https://github.com/urbanairship/ios-library/releases/tag/18.13.1)
+
+### Changes
+- Updated Android SDK to 18.5.1
```

Comment:
I think sometimes we will want to just come up with a different note at the top and link here instead. Could we move the links ?